index.js

```diff
@@ -104,6 +134,32 @@ class PluginXrpPaychan extends PluginBtp {
       }
     }
 
+    // now uses the info protocol to make sure scales are matching
+    if (this._currencyScale !== 6) {
```

Comment:
Should it always check that scales are matching, not only if `this._currencyScale !== 6`?

Reply:
This is there for backwards compatibility; the previous version of the plugin doesn't have the `info` protocol. I _could_ have it send an info request and then just ignore the error if its scale is 6, though.

Reply:
Is backwards compatibility important at this point? If we implement it as a breaking change we can get ride of `this._currencyScale !== 6` and streamline the code a bit.